src/components/Cart/User/SignIn.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="np-sign-in">
+    <h2 class="np-sign-in__title">{{ $t("signIn") }}</h2>
```

Comment:
use `v-t` or `useI18n` hook, check _protip_ on slack - maybe it's not on dev yet @mnieznaj

Reply:
it is on dev now